client/hotrod-client/src/main/java/org/infinispan/client/hotrod/configuration/Configuration.java

```diff
@@ -120,16 +109,17 @@ public class Configuration implements org.infinispan.api.configuration.Configura
    private final int dnsResolverNegativeTTL;
    private final RemoteCacheManagerMetricsRegistry metricRegistry;
    private final int serverFailureTimeout;
+   private long transactionTimeout;
```

Comment:
nitpicking, can be final.

Reply:
I was thinking of making it mutable at runtime

Reply:
Currently, the timeout is only consumed by the `TransactionTable`. 

https://github.com/infinispan/infinispan/blob/0b03a6156c2123ffc40a892494559189b093250b/client/hotrod-client/src/main/java/org/infinispan/client/hotrod/RemoteCacheManager.java#L224-L225

If we want to add this feature, it would make more sense to add a setter in the `RemoteCacheManager`.